.gitignore

```diff
@@ -55,6 +55,7 @@ src/main/webapp/gwt_bree/
 .gradle
 /build/
 /cldr-lib/*.jar
+**/out/
```

Comment:
what are you building to `out/`? i'm not finding anything in this PR that use that path - perhaps should just build to `build/...` or something which is ignored? or is that an old dir or something?

Reply:
it is building to build dir, i am sure where the out dir came from, maybe during some testing, ill double check this